src/main/java/edu/phema/elm_to_omop/io/SpreadsheetWriter.java

```diff
@@ -0,0 +1,37 @@
+package edu.phema.elm_to_omop.io;
+
+import edu.phema.elm_to_omop.helper.Terms;
+import edu.phema.elm_to_omop.vocabulary.phema.PhemaCode;
+import edu.phema.elm_to_omop.vocabulary.phema.PhemaValueSet;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Creates a spreadsheet file on disk
+ */
+public class SpreadsheetWriter {
```

Comment:
Are we using the term "spreadsheet" and "CSV file" interchangeably?

Reply:
Or maybe it's just to be consistent with `SpreadsheetReader`, which actually reads a CSV file? 😔